src/ansys_sphinx_theme/search/fuse_search.py

```diff
@@ -45,19 +58,21 @@ def __init__(self, doc_name, app):
         self.doc_name = doc_name
         self.doc_path = f"{self.doc_name}.html"
         self.env = app.env
+        self.theme_options = app.config.html_theme_options.get("static_search", {})
```

Comment:
This is confusing. Why naming it `theme_options` if you are retrieving the `static_search` dict?